packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -47,6 +47,7 @@ import type Percy from './Percy/Percy.js'
 import { sendStart, sendFinish } from './instrumentation/funnelInstrumentation.js'
 import BrowserStackConfig from './config.js'
 import { setupExitHandlers } from './exitHandler.js'
+import aiSDK from '@browserstack/ai-sdk-node'
```

Comment:
This package doesn't seem to exist yet, I think this should happen first before we merge this in.